cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -358,19 +358,23 @@ public ClusterModel clusterModel(long now,
    * Get the cluster model for a given time window.
    * @param from the start time of the window
    * @param to the end time of the window
-   * @param requirements the model completeness requirement to enforce.
+   * @param minValidPartitionRatio the minimal valid partition ratio requirement of model from request
```

Comment:
1. `minimal` --> `minimum` (the meaning is different).
2. `from request` implies a constraint that this method must be called as part of the user request handling, but it can potentially be used elsewhere. --> would removing this make our intention more clear?